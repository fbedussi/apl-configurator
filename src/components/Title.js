import React from 'react';

const Title = ({currentNode}) => {
    if (!currentNode.title) {
        return null;
    }

    return <h1>{currentNode.title}</h1>
}

export default Title;