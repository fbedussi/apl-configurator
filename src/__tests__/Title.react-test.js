import React from 'react';
import Title from '../components/Title';
import renderer from 'react-test-renderer';

const text = 'Title text';

test('Title component', () => {
  const component = renderer.create(
    <Title text={text} />
  );
  
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});