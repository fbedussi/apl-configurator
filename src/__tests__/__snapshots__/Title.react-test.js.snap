// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component 1`] = `
<h1
  className="title"
>
  Title text
</h1>
`;
